composer.json

```diff
@@ -40,8 +40,8 @@
         "sonata-project/media-bundle": "For using the admin media browser."
     },
     "conflict": {
-        "sonata-project/admin-bundle": "<3.0 || >= 4.0",
-        "sonata-project/media-bundle": "<3.0 || >= 4.0"
+        "sonata-project/admin-bundle": "<3.0 || >=5.0",
```

Comment:
Yep, that's why I wanted to do, but I wasn't sure about it.

Do we change this?